src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -174,6 +175,7 @@ class WiFiManager
     CHIP_ERROR ClearStationProvisioningData();
     CHIP_ERROR Disconnect();
     CHIP_ERROR GetWiFiInfo(WiFiInfo & info) const;
+    const WiFiNetwork & GetWantedNetwork(WiFiNetwork & info) const { return mWantedNetwork; }
```

Comment:
```suggestion
    const WiFiNetwork & GetWantedNetwork() const { return mWantedNetwork; }
```
`info` is not populated

Reply:
Fixed.